src/Sylius/Component/Grid/spec/Data/DataProviderSpec.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Component\Grid\Data;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Component\Grid\Data\DataProvider;
+use Sylius\Component\Grid\Data\DataProviderInterface;
+use Sylius\Component\Grid\Data\DataSourceInterface;
+use Sylius\Component\Grid\Data\DataSourceProviderInterface;
+use Sylius\Component\Grid\Definition\Grid;
+use Sylius\Component\Grid\Filtering\FiltersApplicatorInterface;
+use Sylius\Component\Grid\Parameters;
+use Sylius\Component\Grid\Sorting\SorterInterface;
+use Sylius\Component\Registry\ServiceRegistryInterface;
+
+/**
+ * @mixin DataProvider
+ *
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+class DataProviderSpec extends ObjectBehavior
+{
+    function let(DataSourceProviderInterface $dataSourceProvider, FiltersApplicatorInterface $filtersApplicator, SorterInterface $sorter)
+    {
+        $this->beConstructedWith($dataSourceProvider, $filtersApplicator, $sorter);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Component\Grid\Data\DataProvider');
+    }
+    
+    function it_implements_grid_data_provider_interface()
```

Comment:
I think if it implements ``DataProviderInterface`` spec should be named ``it_implements_data_provider_interface``